src/features/channels/components/channel-list/about/about.tsx

```diff
@@ -42,7 +64,33 @@ const About = ({ title, message, onClick }: Props) => {
       >
         <DialogTitle id="alert-dialog-title">{title}</DialogTitle>
         <DialogContent>
-          <DialogContentText id="alert-dialog-description">{message}</DialogContentText>
+          <DialogContentText id="alert-dialog-description">
+            <Grid className={styles.infoBox}>
+              <Divider />
+
+              <Grid className={styles.infoLine}>
+                <Grid className={styles.infoLineLeft}>{user && user[0].name}</Grid>
+                <Grid className={styles.infoLineRight}>{user && user[0].version}</Grid>
+              </Grid>
```

Comment:
Use the map function to iterate the array. example: user.map((item) => <Grid className={styles.infoLineLeft}>{item.name}</Grid>)

Reply:
Ok, I will